homeassistant/components/risco/__init__.py

```diff
@@ -75,7 +68,11 @@ async def _async_setup_local_entry(hass: HomeAssistant, entry: ConfigEntry) -> b
     data = entry.data
     concurrency = entry.options.get(CONF_CONCURRENCY, DEFAULT_CONCURRENCY)
     risco = RiscoLocal(
-        data[CONF_HOST], data[CONF_PORT], data[CONF_PIN], concurrency=concurrency
+        data[CONF_HOST],
+        data[CONF_PORT],
+        data[CONF_PIN],
+        concurrency=concurrency,
+        communication_delay=1,
```

Comment:
Where did the communication delay come from? That doesn't seem related to the refactor

Reply:
My apologies, this must have sneeked in by accident. This was added during testing of functionality that will be included in another PR.